reference/api/error_codes.md

```diff
@@ -183,8 +183,8 @@ The given `name` is invalid. It should either be a string or `null`.
 
 ### `invalid_task_status`
 
-The requested task status is invalid. Please use one of [these four possible values](/learn/advanced/asynchronous_operations.md#task-status).
+The requested task status is invalid. Please use one of [these four possible values](/reference/api/tasks.md#status).
```

Comment:
Future-proofing: if you don't use an exact number you won't have to change it later when possible values are added or removed.

```suggestion
The requested task status is invalid. Please use one of the [possible values](/reference/api/tasks.md#status).
```